crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/prosecution_concluded/ProsecutionConcludedListenerTest.java

```diff
@@ -72,6 +77,16 @@ void givenJSONMessageIsReceived_whenProsecutionConcludedListenerIsInvoked_thenRe
         softly.assertAll();
     }
 
```

Comment:
localstack provides SQS test queues for integration test. Example can be found in [laa-maat-court-data-api](https://github.com/ministryofjustice/laa-maat-court-data-api/blob/main/maat-court-data-api/src/test/java/gov/uk/courtdata/link/service/CreateLinkListenerEndToEndTest.java)

Reply:
Created a separate story to potentially implement local stack for testing